index.html

```diff
@@ -18,50 +18,49 @@
   <link href="https://fonts.googleapis.com/css?family=Pacifico&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Muli:200,300,400,600" rel="stylesheet">
   <link rel="stylesheet" href="css/styles.css">
-
 </head>
+
 <body>
   <div id="intro">
     <h1>Welcome to The Shade Generator!</h1>
 
-    <input type="number"  max="255" min="0"class="red" placeholder="Red Value"></input>
+    <input type="number"  max="255" min="0"class="red" placeholder="Red Value">
     <br>
-    <input type="number" max="255" min="0" class="green" placeholder="Green Value"></input>
+    <input type="number" max="255" min="0" class="green" placeholder="Green Value">
     <br>
-    <input type="number" max="255" min="0" class="blue" placeholder="Blue Value"></input>
+    <input type="number" max="255" min="0" class="blue" placeholder="Blue Value">
     <br>
-    <button type="button" class="submit">Generate!</button>
+    <button type="button" id="submit">Generate!</button>
   </div>
 
-  <div id="add-js">
+  <div id="add-js"></div>
 
-  </div>
-
-  <script src="js/script.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.1/lib/darkmode-js.min.js"></script>
+  <script src="js/script.js" />
+  <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.1/lib/darkmode-js.min.js" />
   <script>
-  //This uses the cdn above and calls Dark mode and shows widget
-  new Darkmode().showWidget();
-  //These are all you widget options
-  var options = {
-  bottom: '64px', // default: '32px'
-  right: 'unset', // default: '32px'
-  left: '32px', // default: 'unset'
-  time: '0.5s', // default: '0.3s'
-  mixColor: '#fff', // default: '#fff'
-  backgroundColor: '#fff',  // default: '#fff'
-  buttonColorDark: '#100f2c',  // default: '#100f2c'
-  buttonColorLight: '#fff', // default: '#fff'
-  saveInCookies: false, // default: true,
-  label: '🌓', // default: ''
-  autoMatchOsTheme: true // default: true
-  }
-  const darkmode = new Darkmode(options);
-  darkmode.showWidget();
+      //This uses the cdn above and calls Dark mode and shows widget
+      new Darkmode().showWidget();
+      //These are all you widget options
+      var options = {
+          bottom: '64px', // default: '32px'
+          right: 'unset', // default: '32px'
+          left: '32px', // default: 'unset'
+          time: '0.5s', // default: '0.3s'
+          mixColor: '#fff', // default: '#fff'
+          backgroundColor: '#fff',  // default: '#fff'
+          buttonColorDark: '#100f2c',  // default: '#100f2c'
+          buttonColorLight: '#fff', // default: '#fff'
+          saveInCookies: false, // default: true,
+          label: '🌓', // default: ''
+          autoMatchOsTheme: true // default: true
+      }
+
+      const darkmode = new Darkmode(options);
+      darkmode.showWidget();
+  </script>
 
-</script>
   <footer>
-  <p class="footer">Copyright &copy 2019 varlevi</p>
-</footer>
+    <p class="footer">Copyright &copy 2020 varlevi</p>
```

Comment:
Maybe use JavaScript to automatically update this date every year?

Reply:
Yup. Was thinking of that as well, but wanted to keep focus on the refactor only for this PR. Will update in a subsequent PR or just open a new issue with a `good first commit`/`beginners-only` label.

Reply:
Great idea. I’ll open up that issue now.